packages/react/src/QuantityInput/QuantityInput.tsx

```diff
@@ -32,20 +33,24 @@ export function QuantityInput(props: QuantityInputProps): JSX.Element {
           })
         }
       />
-      <TextInput
+      <NumberInput
         id={props.name}
         name={props.name}
         data-testid={props.name + '-value'}
         type="number"
-        step="any"
```

Comment:
Removing `step="any"` makes it integer only.  Not sure what the correct Mantine convention is here.